wagtail/admin/templates/wagtailadmin/workflows/index.html

```diff
@@ -13,49 +13,66 @@ <h1 class="icon icon-{{ view.header_icon }}">
             <div class="right">
                 <div class="addbutton">
                     <a href="{% url view.add_url_name %}" class="button bicolor icon icon-plus">{{ view.add_item_label }}</a>
-                    <a href="{% url 'wagtailadmin_workflows:task_index' %}" class="button">{% trans 'Edit workflow tasks' %}</a>
                 </div>
             </div>
         </div>
     </header>
+
     <div class="nice-padding">
         {% if workflows %}
-            <div id="sites-list">
-                <table class="listing">
-                    <thead>
+            <label>
+                {% if showing_disabled %}
+                    <input type="checkbox" onclick="javascript:window.location.href='.';" checked>
+                {% else %}
+                    <input type="checkbox" onclick="javascript:window.location.href='?show_disabled';">
+                {% endif %}
+
+                {% trans "Show disabled workflows" %}
+            </label>
```

Comment:
2 issues with this:

1. the label gets wrapped weirdly because of the 16% width  rule on it
![Screenshot 2020-06-25 at 14 40 29](https://user-images.githubusercontent.com/31622/85734638-12fc7880-b6f5-11ea-9b55-68ab758c86ae.png)
Changing the text to "Show disabled" sorts this out
2. The inline JS. I suspect it may trip CSP (a la #1288)